analyzers/src/SonarAnalyzer.CSharp/Rules/SqlKeywordsDelimitedBySpace.cs

```diff
@@ -85,39 +84,52 @@ public sealed class SqlKeywordsDelimitedBySpace : SonarDiagnosticAnalyzer
             .OrderBy(i => i)
             .First();
 
-        protected override void Initialize(SonarAnalysisContext context)
-        {
+        protected override void Initialize(SonarAnalysisContext context) =>
             context.RegisterSyntaxNodeActionInNonGenerated(
                 c =>
                 {
                     var namespaceDeclaration = (NamespaceDeclarationSyntax)c.Node;
-                    var compilationUnit = namespaceDeclaration.Parent as CompilationUnitSyntax;
-                    if (compilationUnit == null ||
-                        (!HasSqlNamespace(compilationUnit.Usings) &&
-                        !HasSqlNamespace(namespaceDeclaration.Usings)))
-                    {
-                        return;
-                    }
-                    var visitor = new StringConcatenationWalker(c);
-                    foreach (var member in namespaceDeclaration.Members)
+                    if (namespaceDeclaration.Parent is CompilationUnitSyntax compilationUnit
+                        && (HasSqlNamespace(compilationUnit.Usings)
+                           || HasSqlNamespace(namespaceDeclaration.Usings)))
```

Comment:
this can go to one line I think.
```suggestion
 && (HasSqlNamespace(compilationUnit.Usings)  || HasSqlNamespace(namespaceDeclaration.Usings)))